sysconfig/jureca-booster/compilers.yaml

```diff
@@ -32,9 +32,9 @@ compilers:
     modules: []
     operating_system: centos7
     paths:
-      cc: /usr/local/software/jurecabooster/Stages/2018a/software/icc/2018.2.199-GCC-5.5.0/compilers_and_libraries_2018.2.199/linux/bin/intel64/icc
-      cxx: /usr/local/software/jurecabooster/Stages/2018a/software/icc/2018.2.199-GCC-5.5.0/compilers_and_libraries_2018.2.199/linux/bin/intel64/icpc
-      f77: /usr/local/software/jurecabooster/Stages/2018a/software/ifort/2018.2.199-GCC-5.5.0/compilers_and_libraries_2018.2.199/linux/bin/intel64/ifort
-      fc: /usr/local/software/jurecabooster/Stages/2018a/software/ifort/2018.2.199-GCC-5.5.0/compilers_and_libraries_2018.2.199/linux/bin/intel64/ifort
+      cc: /usr/local/software/jureca/Stages/2019a/software/icc/2019.3.199-GCC-8.3.0/compilers_and_libraries_2019.3.199/linux/bin/intel64/icc
+      cxx: /usr/local/software/jureca/Stages/2019a/software/icc/2019.3.199-GCC-8.3.0/compilers_and_libraries_2019.3.199/linux/bin/intel64/icpc
+      f77: /usr/local/software/jureca/Stages/2019a/software/ifort/2019.3.199-GCC-8.3.0/compilers_and_libraries_2019.3.199/linux/bin/intel64/ifort
+      fc: /usr/local/software/jureca/Stages/2019a/software/ifort/2019.3.199-GCC-8.3.0/compilers_and_libraries_2019.3.199/linux/bin/intel64/ifort
```

Comment:
intel compiler is same for booster or cluster but I am just wondering if they dont now have intel compiler in jurecabooster